style.css

```diff
@@ -66,6 +116,87 @@ h1 span:last-child {
     color: #36b37e;
 }
 
+#contact-box {
+    display: flex;
+    justify-content: center;
+}
+
+.card-container {
+    display: grid;
+    grid-template-columns: 100%;
+    grid-row-gap: 1rem;
+}
+
+.card {
+    border: 1px solid rgb(179, 177, 177);
+    border-radius: 8px;
+    padding-bottom: 1rem;
+}
+
+.card h1 {
+    font-size: 1.5rem;
+    margin: 1rem;
+}
```

Comment:
- [ ] Good job using the correct font size for this `h1`. Please consider giving this `h1` all the required style properties as seen in your design template. Add the properties that you don't have currently.

<img width="141" alt="Loom_dW6QouyEaw" src="https://user-images.githubusercontent.com/81468675/148406920-b15528bd-74ea-465e-8774-54edfd6a6721.png">